### DISCUSSION TITLE
Stop getting error traces leading to generated views

### DISCUSSION THREAD

**Discussion by rolandorojas** [livewire/livewire#3526] [1. Help]
Perhaps I have been doing this wrong forever, but every time I get an error in Livewire, I cannot trace it back to the actual file. 

**Consider the following example.** 
I have an index blade file, in which I call a Livewire component. Let's call it LikeButton. 
Say for example I introduce an error in this `Http\Livewire\LikeButton.php` component. The stack trace will show the error as in something like:

----- Error Trace -----
`ErrorException : Object of class Illuminate\Support\Fluent could not be converted to string (View: \resources\views\index.blade.php)`
This is the actual error and where I am calling the Livewire component, but not where the error actually is. Fine. 

`\storage\framework\views\819a5d7687b601602bea6a69139385a12cc5f7cf.php:34`
This is where I would expect the error to be: the actual Livewire component, but this is actually a generated file with a rendered version of my Livewire file. It is not really where the error is. 

` E:\Projects\Elliptic\Telecial\vendor\livewire\livewire\src\CompilerEngine.php:41`
 `E:\Projects\Elliptic\Telecial\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php:43`
Then you get the rest of the stack trace with the usual lines...
----- Error Trace -----

To me this error trace is useless. Yes, it gives me some idea of how to eventually navigate to the right file, but not where the actual error is. On the contrary, if I introduce an error in a vanilla blade/controller/or other file, the stack trace will take me precisely to the exact line and file where the error is.

Is there a way to get accurate error traces with Livewire? Have I been doing this completely wrong?

Help!

**Comment by ben182** [comment]
This should not be a problem of Livewire since it's just compiled views. Actually, Ignition should handle the error to show the source view file. Can you provide something reproducible?

**Reply by joshhanley** [reply]
@rolandorojas yeah I agree with @ben182, you are getting the trace into the compiled blade view.

If you look at the error using ignition, does it point you to the correct file? How you currently looking at your stack trace, in the log?
